typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/scheduledWorkflowItemDecoratorStrategy.ts

```diff
@@ -0,0 +1,164 @@
+ * %%
+ * Build Tools for VMware Aria
+ * Copyright 2023 VMware, Inc.
+ *
+ * This product is licensed to you under the BSD-2 license (the "License"). You may not use this product except in compliance with the BSD-2 License.
+ *
+ * This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
+ * #L%
+ */
+import { Decorator, MethodDeclaration, SourceFile } from "typescript";
+import { WorkflowItemDescriptor, WorkflowItemType } from "../../../../decorators";
+import { getDecoratorProps } from "../../../helpers/node";
+import CanvasItemDecoratorStrategy from "./canvasItemDecoratorStrategy";
+import { InputOutputBindings, buildItemParameterBindings } from "./helpers/presentation";
+import { findTargetItem } from "../helpers/findTargetItem";
+import { StringBuilderClass } from "../../../../../utilities/stringBuilder";
+import { ScheduledWorkflowItemSourceFilePrinter, SourceFilePrinter } from "./helpers/sourceFile";
+
+/**
+ *
+ * Responsible for printing out the workflow item:
+ * @example
+ * ```xml
+  <workflow-item name="item1" out-name="item0" type="task" launched-workflow-id="9e4503db-cbaa-435a-9fad-144409c08df0">
+	<display-name><![CDATA[Other Workflow]]></display-name>
+	<script encoded="false"><![CDATA[//Auto generated script, cannot be modified !
+var workflowToLaunch = Server.getWorkflowWithId("9e4503db-cbaa-435a-9fad-144409c08df0");
+if (workflowToLaunch == null) {
+	throw "Workflow not found";
+}
+
+var workflowParameters = new Properties();
+workflowParameters.put("first",first);
+workflowParameters.put("second",second);
+scheduledTask = workflowToLaunch.schedule(workflowParameters, workflowScheduleDate);
+]]></script>
+	<in-binding>
+	  <bind name="workflowScheduleDate" type="Date" export-name="workflowScheduleDate"/>
+	  <bind name="first" type="number" export-name="first"/>
+	  <bind name="second" type="number" export-name="second"/>
+	</in-binding>
+	<out-binding>
+	  <bind name="scheduledTask" type="Task"/>
+	</out-binding>
+	<description><![CDATA[Schedule a workflow and create a task.]]></description>
+	<position y="60.0" x="260.0"/>
+  </workflow-item>
+ * ```
+ */
+export default class ScheduledWorkflowItemDecoratorStrategy implements CanvasItemDecoratorStrategy {
+	constructor(private readonly sourceFilePrinter: SourceFilePrinter = new ScheduledWorkflowItemSourceFilePrinter()) { }
+
+	getCanvasType(): string {
+		return "task";
+	}
+
+	getDecoratorType(): WorkflowItemType {
+		return WorkflowItemType.ScheduledWorkflow;
+	}
+
+	registerItemArguments(itemInfo: WorkflowItemDescriptor, decoratorNode: Decorator): void {
+		getDecoratorProps(decoratorNode).forEach((propTuple) => {
+			const [propName, propValue] = propTuple;
+			switch (propName) {
+				case "target":
+					itemInfo.target = propValue;
+					break;
+
+				case "exception":
```

Comment:
I see you didn't declare the `exception` parameter in the `vro-types/vrotsc-annotations/index.d.ts` nor you're handling it in the decorator. You should remove this case if you don't need to handle the `exception` parameter.